src/etl/usecase/anonymiser.py

```
@@ -0,0 +1,142 @@
+    def __init__(self):
+        self.segmenter = Segmenter()
+        self.morph_vocab = MorphVocab()
+        self.emb = NewsEmbedding()
+        self.ner_tagger = NewsNERTagger(self.emb)
+
+        self.mapping = {}
+        self.entities = {}
+        self.counters = {"PER": 1, "LOC": 1, "ORG": 1, "PHONE": 1, "EMAIL": 1, "URL": 1, "USERID": 1}
+
+    def __get_label(self, text, entity_type):
+        if not text: return text
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)